x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/index.tsx

```diff
@@ -83,46 +149,61 @@ export const EndpointDetailsFlyout = memo(() => {
         }),
       });
     }
-  }, [error, toasts]);
+    if (activityError !== undefined) {
+      toasts.addDanger({
+        title: i18n.translate('xpack.securitySolution.endpoint.activityLog.errorTitle', {
+          defaultMessage: 'Could not find activity log for host',
+        }),
+        text: i18n.translate('xpack.securitySolution.endpoint.activityLog.errorBody', {
+          defaultMessage: 'Please exit the flyout and select another host with actions.',
```

Comment:
ℹ️  Seems a bit weird to me that we're instructing the user to go find another host with actions. I don't think that is an activity that the user is normally doing across hosts (or is it? 🤔 😄 ). If they came to the actions log for the host it is because they are investigating this particular host.

cc/ @bfishel

Reply:
Indeed. This is going to be part of the empty state and the not-found details banner that we saw the other day. Not sure if we've nailed it down yet. So the idea is if we don't have a host then we also won't have host actions log.

![Screenshot 2021-06-02 at 14 49 00](https://user-images.githubusercontent.com/1849116/120482733-b2b89100-c3b1-11eb-8d6c-afb7b3ef171b.png)

Here's the link https://www.figma.com/file/BeTJy7bpHRPtHiNJtWU7Pj/OLM-7.12%2B?node-id=4476%3A0